tests/unit/validate-identity.test.ts

```diff
@@ -18,7 +18,7 @@ describe("validateIdentity unit test", () => {
     expect.assertions(2);
 
     const mockedFetchResponse = new Response(
-      JSON.stringify({ error: "INVALID" }),
+      JSON.stringify({ code: "invalid_validation_code" }),
```

Comment:
what about importing them from the errors module?

Reply:
Good point